modules/nw-multi-network-policy-enable.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_networks/configuring-multi-network-policy.adoc
+
+[id="nw-multi-network-policy-enable_{context}"]
+= Enabling multi-network policy for the cluster
+
+As a cluster administrator, you can enable multi-network policy support on your cluster.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a file with the following YAML:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  useMultiNetworkPolicy: true
+----
+
+. To enable multi-network policy, enter the following command. Replace `<name>` with the name of the YAML file you created in the previous step.
```

Comment:
Maybe "Configure the cluster to enable multi-network policy:"

I believe you stated a preference for "To <goal>, enter..." but that ends up with sentence structure that isn't parallel with the first step.  I also suspect that you can drop the "Replace..." sentence even if you don't choose to be prescriptive.